test/tests/selftest/pbs_managers_operators.py

```diff
@@ -38,25 +38,42 @@
 from tests.selftest import *
 
 
-class ManagersOperators(TestSelf):
+class TestManagersOperators(TestSelf):
 
     """
-    This test suite contains tests related to managers and operators
-
+        Additional managers users, except current user and MGR_USER
+        should get unset after test setUp run
     """
     def test_managers_unset_setup(self):
         """
-        Additional managers users, except current user should get unset
-        after test setUp run
+        Additional managers users, except current user and MGR_USER should
+        get unset after test setUp run
         """
         runas = ROOT_USER
+        manager_usr_str = str(MGR_USER) + '@*'
         current_usr = pwd.getpwuid(os.getuid())[0]
-        attr = {ATTR_managers: current_usr + '@*'}
-        self.server.expect(SERVER, attrib=attr)
+        current_usr_str = str(current_usr) + '@*'
+        mgr_users = {manager_usr_str, current_usr_str}
+        svr_mgr = self.server.status(SERVER, 'managers')[0]['managers']\
+            .split(",")
+        self.assertEqual(mgr_users, set(svr_mgr))
```

Comment:
you are comparing a dict with a set ? I think you need to make them both sets.

Reply:
both are sets. I initialized the set with  ->  set = {'str a', 'str b'}

Reply:
ah, my bad, thanks for explaining